### DISCUSSION TITLE
[ FF7 ] Not working

### DISCUSSION THREAD

**Discussion by epshecateix** [julianxhokaxhiu/FFNx#180] [General]
Hello, I am not sure how to report this, When I tried to play FF7 using the 7thH mod every time I start it error happened and I am not sure how to fix it. 

[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/5549935/FFNx.log)

**Comment by eve-atum** [comment]
Turn off the 60fps option in the Animations mod.

**Comment by epshecateix** [comment]
> Turn off the 60fps option in the Animations mod.

thank you very much I really appreciate it.
